deployment/helm/templates/deployment.yaml

```diff
@@ -66,6 +66,11 @@ spec:
             value: "{{ .Values.trusty.endpoint }}"
           - name: "SIGSTORE_NO_CACHE"
             value: "true"
+          - name: "MINDER_DB_PASSWORD"
```

Comment:
I think there is a bit of confusion in how the variables are transformed into config values. I haven't tried this PR, but the way I read this hunk is that this would set the equivalent of what a `db.password` in config would set, wouldn't it? At least that's how I think `SetViperDefaults` works - you'd strip the MINDER prefix and then substitute underscores for dots. Note that we set `MINDER_IDENTITY_SERVER_CLIENT_SECRET_FILE` in this file as an example.

And as far as datebase goes, I don't see a `db` field in the `Config` struct at all, but a `database` field instead, so if you want to set the DB password, you'd set `MINDER_DATABASE_PASSWORD`.

Reply:
Thanks! Remove env changes here, there will be a follow-up PR in infra for the rest of the configuration: https://github.com/stacklok/infra/pull/319

The name will be ```MINDER_EVENTS_SQL_CONNECTION_DBPASS``` to reflect the nested struct structure we'll have.